UnicornRender/video/source/vulkan/VkMesh.cpp

```diff
@@ -98,22 +98,22 @@ uint32_t VkMesh::IndicesSize() const
 
 bool VkMesh::IsColored() const
 {
-    return m_mesh.GetMaterial().IsColored();
+    return m_mesh.GetMaterial()->IsColored();
```

Comment:
Now that I think of it - is there a point to not just create `GetMaterial` so user code can call all these methods from there? Why proxy everything?